contracts/token/psp20/traits.rs

```diff
@@ -1,11 +1,11 @@
 use ink_prelude::{string::String};
 use brush::traits::{AccountId, Balance};
 
-/// The ERC-20 error type. Contract will assert one of this errors.
+/// The PSP-20 error type. Contract will assert one of this errors.
 #[derive(strum_macros::AsRefStr)]
-pub enum Erc20Error {
+pub enum PSP20Error {
     /// Unknown error type for cases if writer of traits added own restrictions
-    Unknown(String),
+    Unknown(&'static str),
```

Comment:
why change to &'static str?

Reply:
Because it takes less memory=) And we know errors during compilation time